Packs/CortexAttackSurfaceManagement/Playbooks/Cortex_ASM_-_ASM_Alert.yml

```
@@ -2798,7 +2578,7 @@ tasks:
           right:
             value:
               simple: Prompt all options
-    - label: Prompt no AR
+    - label: ' Prompt no AR'
```

NIT, remove white space.
```suggestion
    - label: 'Prompt no AR'
```

---

took care of this in https://github.com/demisto/content/pull/24786/commits/51d1b222ebaab7f0349d9ea70bac69cab57c96ab